Model/Repository.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Ziffity\Task\Model;
+
+use Ziffity\Task\Api\RepositoryInterface;
+use Ziffity\Task\Api\Data;
+use Ziffity\Task\Model\ResourceModel\Custom as ResourceCustomCart;
+use Ziffity\Task\Model\ResourceModel\Custom\CollectionFactory as CustomCartCollectionFactory;
+use Magento\Framework\Api\DataObjectHelper;
+use Magento\Framework\Api\SearchCriteria\CollectionProcessorInterface;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Exception\CouldNotDeleteException;
+use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Reflection\DataObjectProcessor;
+
+
+
+class Repository implements RepositoryInterface
+{
+   
+    protected $resource;
+
```

Comment:
@NandhiniKarthi Please format the code across the modules. We shouldn't leave this many white spaces between lines.